shim.js

```diff
@@ -1,3 +1,11 @@
+// The XMLHttpRequest is required since the AlogliaClient use it globally
+// Since `react-native@0.49.x` the `InitializeCore` module is not executed anymore
+// and so the polyfills are not applied in Jest environment.
+// see on 0.48.4: https://github.com/facebook/react-native/blob/v0.48.4/jest/setup.js#L37
+// see on 0.49.x: https://github.com/facebook/react-native/blob/v0.49.0/jest/setup.js#L37
+// eslint-disable-next-line
+global.XMLHttpRequest = require('XMLHttpRequest');
```

Comment:
where's this shim coming from then 🤔 I don't see it in the devDeps

Reply:
that probably caused the test failure: https://travis-ci.org/algolia/react-instantsearch/builds/301352306#L581